src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/RcwTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Marshalling;
+using Xunit;
+using Xunit.Sdk;
+
+namespace ComInterfaceGenerator.Tests;
+
+[GeneratedComInterface]
+[InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+[Guid("2c3f9903-b586-46b1-881b-adfce9af47b1")]
+public partial interface IComInterface1
+{
+    int GetData();
+    void SetData(int n);
+}
+
+internal sealed unsafe partial class MyGeneratedComWrappers : StrategyBasedComWrappers
+{
+    protected sealed override unsafe ComInterfaceEntry* ComputeVtables(object obj, CreateComInterfaceFlags flags, out int count) => throw new UnreachableException("Not creating CCWs yet");
+}
+
+public static unsafe partial class Native
+{
+    [LibraryImport(NativeExportsNE.NativeExportsNE_Binary, EntryPoint = "get_com_object")]
+    public static partial void* NewNativeObject();
+}
+
+
+public class RcwTests
+{
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/83289")]
```

Comment:
```suggestion
    [ActiveIssue("https://github.com/dotnet/runtime/issues/83289")]
    [Fact]
```

Now that we've merged in the support for the right signatures, we should merge/rebase and enable this test.